apps/console/src/features/applications/components/forms/inbound-oidc-form.tsx

```diff
@@ -89,6 +89,19 @@ export const InboundOIDCForm: FunctionComponent<InboundOIDCFormPropsInterface> =
     const [ showRegenerateConfirmationModal, setShowRegenerateConfirmationModal ] = useState<boolean>(false);
     const [ showRevokeConfirmationModal, setShowRevokeConfirmationModal ] = useState<boolean>(false);
     const [ allowedOrigins, setAllowedOrigins ] = useState("");
+    const [
+        isTokenBindingTypeSelected,
+        setIsTokenBindingTypeSelected
+    ] = useState<boolean>(false);
+
+    /**
+     * Sets if a valid token binding type is selected.
+     */
+    useEffect(() => {
+        if (initialValues.accessToken.bindingType !== "None") {
+            setIsTokenBindingTypeSelected(true);
+        }
+    }, [ initialValues.accessToken.bindingType ]);
```

Comment:
Incase if `accessToken` is undefined the app will break. Would be better to null check here.

```suggestion
    }, [ initialValues?.accessToken?.bindingType ]);
```

Reply:
Resolved with 4af7f39